docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,555 @@
+mp4Src: "/tutorials/img/bedtime-songs-bot/robot_babysitter.mp4"
+videoAlt: "A demonstration of the bedtime songs bot is taking place in an office. Tess holds up brightly colored puzzle pieces in front of the camera of a Macbook laptop. As the webcam on the laptop recognizes the puzzle pieces, different songs start to play on the speakers of the computer."
+authors: [ "Tess Avitabile", "Sierra Guequierre" ]
+languages: [ "python" ]
+viamresources: [ "camera", "sensor", "mlmodel", "vision" ]
+level: "Beginner"
+date: "21 April 2023"
+cost: "0"
+---
+
+## Author's Note
+
+**"When I started at Viam, Eliot Horowitz told me the best way to test the product is to try to automate something I do in my life with a robot.
+As a parent of a 3-year-old and a 1-year-old, I am often presented with a toy and asked to sing a song about it.
+When I was testing out Viam's ML Model service, I came up with the idea of using machine learning to make my computer do this simple task for my kids when I'm not around."**
+
+- Tess, Engineering Director at Viam
+
+Follow this tutorial to train a machine learning model to make your own "bedtime songs bot" out of a personal computer.
+
+## Get Started
+
+To make your own singing robot, you need the following hardware:
+
+- A computer with a webcam, speakers, and the [Go Client SDK](https://pkg.go.dev/go.viam.com/rdk) installed.
+  Tess used a Macbook, but you can use any PC with a Viam-compatible operating system that meets the above requirements.
+
+While following this tutorial, you'll complete the following steps to train and utilize a machine learning model on a webcam:
+
+1. [**Train your ML Model with pictures of toys:**](#train-your-ml-model-with-pictures-of-toys)
+
+- [Configure the camera on your laptop to capture data](#configure-your-webcam-to-capture-data) with the Data Management service and use it to capture images.
+   After that, filter your image data with [tags](#tag-data) and use it to train a [machine learning model](/services/ml/) on [the Viam app](https://app.viam.com).
+
+2. [**Use your ML Model to sing songs to your kids:**](#use-your-ml-model-to-sing-songs-to-your-kids)
+
+- [Configure your webcam to act as a shape classifier](#configure-your-webcam-to-act-as-a-shape-classifier) with Viam's [ML Model](/services/ml/) and [Vision](/services/vision/) Services, [record bedtime songs](#record-bedtime-songs), and [program the robot](#program-your-bedtime-songs-bot) to play a song when it recognizes a shape.
+
+## Train your ML Model with pictures of toys
+
+### Configure your webcam to capture data
+
+In the [Viam app](https://app.viam.com), create a new robot and follow the steps on your new robot’s **Setup** tab.
+
+Navigate to your robot's page on the app and click on the [**Config** tab](/manage/configuration/).
+
+First, add your personal computer's webcam to your robot as a [camera](/components/camera/) by creating a new component with **type** `camera` and **model** `webcam`:
```

Comment:
This does apply to both the config builder and json template but it's not as clear as it is for example [here](https://docs.viam.com/tutorials/projects/claw-game/#configure-the-board)

Reply:
Ok I moved the info into config builder!